From 16f716eadd61be80b28839396105c020e401294c Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Wed, 20 Apr 2011 14:32:06 +0200
Subject: [PATCH] EMU10K1.conf: Don't lock hook controls

Locking the controls in hooks causes a BUSY error when multiple
streams are opened, typically in PulseAudio.  Better not to touch
the locking in hooks in such a case.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 src/conf/cards/EMU10K1.conf |   20 ++++++++++----------
 1 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/src/conf/cards/EMU10K1.conf b/src/conf/cards/EMU10K1.conf
index 956e163..b537864 100644
--- a/src/conf/cards/EMU10K1.conf
+++ b/src/conf/cards/EMU10K1.conf
@@ -21,7 +21,7 @@ EMU10K1.pcm.front.0 {
 				interface PCM
 				name "EMU10K1 PCM Send Volume"
 				index { @func private_pcm_subdevice }
-				lock true
+				# lock true
 				optional true
 				value [ 255 255 0 0 255 0 0 0 0 255 0 0 ]
 			}
@@ -29,7 +29,7 @@ EMU10K1.pcm.front.0 {
 				# for compatibility with older drivers
 				name "EMU10K1 PCM Send Volume"
 				index { @func private_pcm_subdevice }
-				lock true
+				# lock true
 				optional true
 				value [ 255 255 0 0 255 0 0 0 0 255 0 0 ]
 			}
@@ -37,7 +37,7 @@ EMU10K1.pcm.front.0 {
 				interface PCM
 				name "EMU10K1 PCM Send Routing"
 				index { @func private_pcm_subdevice }
-				lock true
+				# lock true
 				optional true
 				value [ 8 9 0 0 8 9 0 0 8 9 0 0 ]
 			}
@@ -45,7 +45,7 @@ EMU10K1.pcm.front.0 {
 				# for compatibility with older drivers
 				name "EMU10K1 PCM Send Routing"
 				index { @func private_pcm_subdevice }
-				lock true
+				# lock true
 				optional true
 				value [ 8 9 0 0 8 9 0 0 8 9 0 0 ]
 			}
@@ -72,7 +72,7 @@ EMU10K1.pcm.rear.0 {
 				interface PCM
 				name "EMU10K1 PCM Send Volume"
 				index { @func private_pcm_subdevice }
-				lock true
+				# lock true
 				optional true
 				value [ 0 0 255 255 0 0 255 0 0 0 0 255 ]
 			}
@@ -80,7 +80,7 @@ EMU10K1.pcm.rear.0 {
 				# for compatibility with older drivers
 				name "EMU10K1 PCM Send Volume"
 				index { @func private_pcm_subdevice }
-				lock true
+				# lock true
 				optional true
 				value [ 0 0 255 255 0 0 255 0 0 0 0 255 ]
 			}
@@ -133,7 +133,7 @@ EMU10K1.pcm.center_lfe.0 {
 				interface PCM
 				name "EMU10K1 PCM Send Volume"
 				index { @func private_pcm_subdevice }
-				lock true
+				# lock true
 				optional true
 				value [ 255 255 0 0 255 0 0 0 0 255 0 0 ]
 			}
@@ -141,7 +141,7 @@ EMU10K1.pcm.center_lfe.0 {
 				# for compatibility with older drivers
 				name "EMU10K1 PCM Send Volume"
 				index { @func private_pcm_subdevice }
-				lock true
+				# lock true
 				optional true
 				value [ 255 255 0 0 255 0 0 0 0 255 0 0 ]
 			}
@@ -149,7 +149,7 @@ EMU10K1.pcm.center_lfe.0 {
 				interface PCM
 				name "EMU10K1 PCM Send Routing"
 				index { @func private_pcm_subdevice }
-				lock true
+				# lock true
 				optional true
 				value [ 6 7 0 0 6 7 0 0 6 7 0 0 ]
 			}
@@ -157,7 +157,7 @@ EMU10K1.pcm.center_lfe.0 {
 				# for compatibility with older drivers
 				name "EMU10K1 PCM Send Routing"
 				index { @func private_pcm_subdevice }
-				lock true
+				# lock true
 				optional true
 				value [ 6 7 0 0 6 7 0 0 6 7 0 0 ]
 			}
-- 
1.7.6

